src/tools/dev/scripts/bv_support/bv_advio.sh

```diff
@@ -185,6 +185,12 @@ function build_advio
     sed "s%FileIO IDL DocIO%FileIO DocIO%g" configure > c2
     mv c2 configure
     chmod 750 ./configure
+    # configure has a number of broken try-run code-blocks
+    if [[ "$OPSYS" == "Darwin" ]]; then
+        sed -i orig -e 's/^main()/int main()/' configure
+    else
+        sed -i.orig -e 's/^main()/int main()/' configure
+    fi
     info "Invoking command to configure AdvIO"
```

Comment:
Many try-run blocks for sizes of type in configure defined a `mainI()` with no type which causes modern compilers to fail outright. This `sed` command fixes them. Its a little different on macOS then other *nix systems. Windows?

Reply:
@markcmiller86 Sorry, didn't see the question about Windows sooner.  Not using the build_visit script on Windows, so not an issue.